### DISCUSSION TITLE
Pydantic V2 debian package

### DISCUSSION THREAD

**Discussion by MarkBlackbird** [pydantic/pydantic#9273] [Question] [answered]
I tried packaging my app as debian package, but when started as such it started throwing tantrum about being unable to load field_serializer from pydantic. 

Since then I have figured out that there is simply no debian package for v2 (which I'm using) and python3-pydantic installs v1 instead, which is causing the error I'm seeing.

I'm wondering if there is anything I can do to resolve this other than downgrading code to use v1? 

This issue provides some reference: https://github.com/pydantic/pydantic/issues/8416 but does not provide either eta for v2 to become debian package, nor how to resolve this issue, nor is bug it is linking to valid for v2 (although it mentions it, comments imply that v2 won't be done and pydantic is marked as Fix Released so no more work will take place as part of this ticket).

**Comment by samuelcolvin** [comment]
Thanks for the question, I took would love pydantic V2 to be on Debian main.

I'll email my contact at canonical now and ask them if there's any plan or progress.

**Reply by MarkBlackbird** [reply]
@samuelcolvin any answer from Canonical?

**Comment by cjwatson** [comment, marked as answer]
Debian testing and the current Ubuntu development release now both have pydantic 2.8.2, so pydantic v2 will be in both Debian 13 and Ubuntu 24.10.

Sorry this took so long; it took some time to get all the new Rust pieces lined up properly and sort out various bits of CI.  (I just did the last push to get it over the line; other people did most of the work.)

**Reply by cjwatson** [reply]
(And yes, upgrading to the actual current release is next on the list ...)
